### PR TITLE
update Sauce Connect documentation

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: dashboard/test/ui/README.md [hunk 1]
```diff
@@ -46,7 +46,10 @@ You can find the values for these settings in your saucelabs account settings (`
 If you want to run tests on saucelabs against localhost you need to set up your tunnel:
 
 1. Login to Saucelabs and download the [tunnel](https://app.saucelabs.com/tunnels)
-2. Start the tunnel via `~/bin/sc/ -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`. Notes: The link above has an example command line with your credentials that you can copy. `~/bin/sc/` should be replaced with the path that points to the tunnel software downloaded in step 1.
+2. Start the tunnel via `~/bin/sc/ -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`
```
Comment (maddiedierker) [inline dashboard/test/ui/README.md]: nit/typo that i think i introduced 😅 
```suggestion
2. Start the tunnel via `~/bin/sc/ -u $SAUCELABS_USERNAME -k $SAUCELABS_ACCESS_KEY`
```

File: dashboard/test/ui/README.md [hunk 2]
```diff
@@ -46,7 +46,10 @@ You can find the values for these settings in your saucelabs account settings (`
...
+   - Notes: The link above has an example command line with your credentials that you can copy.
+     - `bin/sc` should be replaced with the path that points to the tunnel software downloaded in step 1.
```
Comment (maddiedierker) [inline dashboard/test/ui/README.md:51]:
```suggestion
     - `~bin/sc/` should be replaced with the path that points to the tunnel software downloaded in step 1.
```

File: dashboard/test/ui/README.md [hunk 3]
```diff
@@ -46,7 +46,10 @@ You can find the values for these settings in your saucelabs account settings (`
...
+     - The unique tunnel identifier `-i [tunnel_id]` included in that command can be safely ignored; if you leave it in, you'll also need to set the `tunnelIdentifier` option in the `sauce_capabilities` config. See [Using Sauce Connect Tunnel Identifiers](https://wiki.saucelabs.com/display/DOCS/Using+Sauce+Connect+Tunnel+Identifiers#UsingSauceConnectTunnelIdentifiers-TheBasicsofUsingTunnelIdentifiers) for more details.
```
Comment (maddiedierker) [inline dashboard/test/ui/README.md]:
```suggestion
     - The unique tunnel identifier `-i $TUNNEL_ID` included in that command can be safely ignored; if you leave it in, you'll also need to set the `tunnelIdentifier` option in the `sauce_capabilities` config. See [Using Sauce Connect Tunnel Identifiers](https://wiki.saucelabs.com/display/DOCS/Using+Sauce+Connect+Tunnel+Identifiers#UsingSauceConnectTunnelIdentifiers-TheBasicsofUsingTunnelIdentifiers) for more details.
```

**Review by maddiedierker — APPROVED** [review]
yay, thank you so much for this! i left a few nits as "suggested changes" but this LGTM

File: dashboard/test/ui/README.md (hunk 2 above)
Comment (Hamms) [reply dashboard/test/ui/README.md:51]: the example command is now `bin/sc`, not `~bin/sc/`; I updated this to match

File: dashboard/test/ui/README.md (hunk 3 above)
Comment (Hamms) [reply dashboard/test/ui/README.md]: this implies to me that folks have set TUNNEL_ID as a bash environment variable, which I don't think we want to imply

Comment (maddiedierker) [reply dashboard/test/ui/README.md]: yeah, that makes sense -- i really only care that we're consistent with the original command: `bin/sc -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`

Comment (maddiedierker) [reply dashboard/test/ui/README.md]: so we could change the original command to `bin/sc -u [saucelabs_username] -k [saucelabs_access_key]` instead?

**Pull request merged**